docs/models/gqi.rst

```diff
@@ -4,6 +4,42 @@
 Generalized q-Sampling Imaging (GQI)
 ####################################
 
+GQI is a model-free approach that estimates water
+diffusion orientation distribution functions (dODFs) using an analytic transform of
+the diffusion signal :cite:p:`yeh2010`.
+Like the other models, GQI produces a number
+of parametric microstructure maps such as generalized fractional anisotropy (GFA),
+quantitative anisotropy (QA), and isotropic component (ISO) :cite:p:`yeh2013`.
+
+
+GQI Foundational Papers
+~~~~~~~~~~~~~~~~~~~~~~~
+
+- Generalized q-Sampling Imaging (GQI) was introduced as an analytical, model-free diffusion MRI reconstruction method that bridges Q-ball imaging (single-shell HARDI) and diffusion spectrum imaging (grid sampling) :cite:p:`yeh2010`. It estimates the spin distribution function (SDF), an ODF-like representation of diffusion directions, from a variety of sampling schemes.
```

Comment:
Again, just noting that we probably want to go over all of this to make sure that similar formatting choices are made in all of these with respect to bullet points, section headers, use of bold-face, etc.